packages/flutter/lib/src/material/icon_theme_data.dart

```diff
@@ -14,8 +21,10 @@ class IconThemeData {
   /// An opacity to apply to both explicit and default icon colors.
```

Comment:
I filed https://github.com/flutter/flutter/issues/3273 about this issue.  I suspect we need this at a lower level of the system so we can catch all animation value -> opacity values.  /cc @HansMuller